openadapt/crud.py

```diff
@@ -1,16 +1,13 @@
-from loguru import logger
+import io
+
+import mss.tools
 import sqlalchemy as sa
+from loguru import logger
 
-from openadapt.db import Session
-from openadapt.models import (
-    ActionEvent,
-    Screenshot,
-    Recording,
-    WindowEvent,
-    PerformanceStat,
-    MemoryStat
-)
 from openadapt.config import STOP_SEQUENCES
+from openadapt.db import Session
+from openadapt.models import (ActionEvent, MemoryStat, PerformanceStat,
+                              Recording, Screenshot, WindowEvent)
 
```

Comment:
This is linted by `isort` via `precommit-hook

Reply:
@Mustaballer can we please modify the default `isort` config to format this as `black` would (i.e. how it was before)? If not, can we add  a pre-commit hook to run black as well?